commit a47451353423913e65ad7f3260606cf49420b00f
Author: Ping-Ke Shih <pkshih@realtek.com>
Date:   Wed Dec 28 15:40:04 2016 -0600

    rtlwifi: Fix alignment issues
    
    commit 40b368af4b750863b2cb66a3a9513241db2f0793 upstream.
    
    The addresses of Wlan NIC registers are natural alignment, but some
    drivers have bugs. These are evident on platforms that need natural
    alignment to access registers.  This change contains the following:
     1. Function _rtl8821ae_dbi_read() is used to read one byte from DBI,
        thus it should use rtl_read_byte().
     2. Register 0x4C7 of 8192ee is single byte.
    
    Signed-off-by: Ping-Ke Shih <pkshih@realtek.com>
    Signed-off-by: Larry Finger <Larry.Finger@lwfinger.net>
    Signed-off-by: Kalle Valo <kvalo@codeaurora.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1328_a47451353423913e65ad7f3260606cf49420b00f.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1328_a47451353423913e65ad7f3260606cf49420b00f.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
